src/lib/test/ObjectTests.cpp

```diff
@@ -1975,12 +2103,12 @@ void ObjectTests::testAllowedMechanisms()
 
 	CK_KEY_TYPE keyType = CKK_GENERIC_SECRET;
 	CK_OBJECT_CLASS secretClass = CKO_SECRET_KEY;
-	CK_BYTE key[64];
+	CK_BYTE key[65] = { "0000000000000000000000000000000000000000000000000000000000000000" };
```

Comment:
This looks like some separate change unrelated to the actual issue. Can you put it into separate commit?

Reply:
This was to fix a Valgrind memory error so that there were no issues when running the ObjectTests. Have split this into two commits.